cirq/contrib/json.py

```diff
@@ -0,0 +1,14 @@
+"""Contains functions for adding JSON serialization and de-serialization for
+classes in Contrib.
+
+"""
+
+
```

Comment:
you could add
```python
DEFAULT_RESOLVERS = [contrib_class_resolver] + protocols.json.DEFAULT_RESOLVERS]
```

then

```python
import cirq
import cirq.contrib
data = cirq.read_json(fn, resolvers=cirq.contrib.DEFAULT_RESOLVERS)
```

The above relies on python namespacing for disambiguation. You could make the name more specific: `DEFAULT_CONTRIB_RESOLVERS` if you want

Reply:
Good idea, done and used in the test.